internal/pkg/cli/env_init.go

```diff
@@ -172,46 +189,137 @@ func (o *initEnvOpts) Execute() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateCustomizedResources() error {
```

Comment:
nit: `validateCustomizedVPC`

Reply:
I was planning to make it more extendable so that when we have more types of customized resources available, we can add the validation in this function. What do you think?

Reply:
yeah, I wasn't sure. I personally like naming things with what they represent now if I'm not certain how it'll evolve. But this is perfectly fine and understandable.